newIDE/app/src/PropertiesEditor/index.js

```diff
@@ -9,16 +9,27 @@ import MenuItem from 'material-ui/MenuItem';
 import Edit from 'material-ui/svg-icons/image/edit';
 import IconButton from 'material-ui/IconButton';
 
+import {
+  type ResourceSource,
+  type ChooseResourceFunction,
+} from '../ResourcesList/ResourceSource.flow';
+import { type ResourceExternalEditor } from '../ResourcesList/ResourceExternalEditor.flow';
+
 export type Instance = Object; // This could be improved using generics.
 export type Instances = Array<Instance>;
 export type Field = Object;
 export type Schema = Array<Field>;
+export type Project = Object;
 
 type Props = {|
   onInstancesModified?: Instances => void,
   instances: Instances,
   schema: Schema,
   mode?: 'column' | 'row',
+  project?: Project,
```

Comment:
use `gdProject` :) Typing for objects of libGD.js is done inside `flow-typed/libGD.js` file. Most of the definitions right now are very simple, but it's important to always use the proper type, because in the future we might want to write them properly or autogenerate them from C++ classes.